Source/monstdat.h

```diff
@@ -89,44 +89,44 @@ enum class MonsterAvailability : uint8_t {
 };
 
 struct MonsterData {
-	const char *mName;
-	const char *GraphicType;
-	const char *sndfile;
-	const char *TransFile;
+	const char *name;
+	const char *graphicType;
+	const char *sndFile;
+	const char *transFile;
 	MonsterAvailability availability;
 	uint16_t width;
-	uint16_t mImage;
-	bool has_special;
-	bool snd_special;
-	int8_t Frames[6];
-	int8_t Rate[6];
-	int8_t mMinDLvl;
-	int8_t mMaxDLvl;
-	int8_t mLevel;
-	uint16_t mMinHP;
-	uint16_t mMaxHP;
-	_mai_id mAi;
+	uint16_t image;
+	bool hasSpecial;
+	bool sndSpecial;
+	int8_t frames[6];
+	int8_t rate[6];
+	int8_t minDunLvl;
+	int8_t maxDunLvl;
+	int8_t level;
+	uint16_t maxHitPointsMinimum;
+	uint16_t maxHitPointsMaximum;

```

Comment:
is `max` really required at the start of these? since this is the basis for the creation of monsters, isn't it implied that this structure is not used to store current hitpoints?

Reply:
I agree with removing max.

We already settled on hitPoints in a previous cleanup. HP isn't good for searching either.

Reply:
I put `max` there to distinguish between field `maxHitPoints` in `Monster` and this one. Without `max` they would be similarly named, but they don't have a `=` correspondence. Those two vars in `MonsterData` are used for generating `maxHitPoints` for a given monster, so I tried to reflect that in those names. I think the name `hitPointsMaximum` could wrongly lead to an assumption that this field and the `maxHitPoints` field in monster are the same. 
 
 But maybe there are better names for those 2 altogether...

Reply:
I think they are two separeret contexts